### PR TITLE
Paper wallet dark theme

### PR DESCRIPTION
**Pull request by Nebyt**

### CODE REVIEW

File: packages/yoroi-extension/app/components/common/Autocomplete.js [hunk 1]
```diff
@@ -177,10 +175,7 @@ function Autocomplete({
             })}
           />
         </Box>
-        <CheckWrapper>
-          {done === true ? <SuccessIcon /> : null}
-          {Boolean(error) === true ? <ErrorIcon /> : null}
```
Comment (nistadev) [inline packages/yoroi-extension/app/components/common/Autocomplete.js:182]: There's no error icon anymore?

File: packages/yoroi-extension/app/components/transfer/BaseTransferPage.js [hunk 2]
```diff
@@ -59,28 +55,35 @@ export default class BaseTransferPage extends Component<Props> {
         actions={actions}
         closeOnOverlayClick={false}
         onClose={onBack}
-        className={styles.dialog}
+        className={undefined}
```
Comment (nistadev) [inline packages/yoroi-extension/app/components/transfer/BaseTransferPage.js]: If undefined, just remove the entire prop.

File: packages/yoroi-extension/app/components/transfer/cards/TransferTypeSelect.js [hunk 3]
```diff
@@ -21,38 +22,43 @@ const messages = defineMessages({
...
 
   render(): Node {
     const { intl } = this.context;
+    const faqLink = (
+      <Link
+        href={intl.formatMessage(globalMessages.faqLinkUrl)}
+        onClick={event => handleExternalLinkClick(event)}
+        id="settings:support-faq-link"
+      >
+        FAQ
```
Comment (nistadev) [inline packages/yoroi-extension/app/components/transfer/cards/TransferTypeSelect.js]: Use intl

File: packages/yoroi-extension/app/components/common/Autocomplete.js (hunk 1 above)
Comment (Nebyt) [reply packages/yoroi-extension/app/components/common/Autocomplete.js:182]: It is removed from all inputs in design.

File: packages/yoroi-extension/app/components/transfer/cards/TransferTypeSelect.js (hunk 3 above)
Comment (Nebyt) [reply packages/yoroi-extension/app/components/transfer/cards/TransferTypeSelect.js]: Fixed

File: packages/yoroi-extension/app/components/transfer/BaseTransferPage.js (hunk 2 above)
Comment (Nebyt) [reply packages/yoroi-extension/app/components/transfer/BaseTransferPage.js]: Fixed

**Review by nistadev — APPROVED** [review]
Good job 🙌🏽

**Pull request merged**